functests/performance/performance.go

```diff
@@ -170,8 +173,60 @@ var _ = Describe("performance", func() {
 			execSysctlOnWorkers(workerRTNodes, sysctlMap)
 		})
 	})
+
+	Describe("Updating parameters in performance profile", func() {
+		var profile *performancev1alpha1.PerformanceProfile
+
+		BeforeEach(func() {
+			profile, err = profiles.GetByNodeLabels(
+				testclient.Client,
+				map[string]string{
+					fmt.Sprintf("%s/%s", testutils.LabelRole, testutils.RoleWorkerRT): "",
+				},
+			)
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It("Verifies that hugepages can be updated thru performance profile", func() {
```

Comment:
Let's add the test case id's and other parameters while we are at it.

Reply:
Will do, but at the end this test should combine several polarion tests.. do you know how it's better to mark all of them?